### PR TITLE
chore: use hsl for tailwind and shadcn theming

### PR DESCRIPTION
**Pull request by BrunoQuaresma**

### CODE REVIEW

File: site/tailwind.config.js [hunk 1]
```diff
@@ -8,7 +8,8 @@ module.exports = {
 	important: ["#root", "#storybook-root"],
 	theme: {
 		fontSize: {
-			"2xs": ["0.626rem","0.875rem"],
+			"2xs": ["0.626rem", "0.875rem"],
+			xs: ["0.75rem", "1rem"],
```
Comment (jaaydenh) [inline site/tailwind.config.js:12]: "0.75rem", "1rem" is already the Tailwind default for xs so this should not be necessary

**Comment by jaaydenh** [comment]
@BrunoQuaresma just to confirm since there is no description. HSL is needed to get the alpha for colors to work correctly?

File: site/tailwind.config.js (hunk 1 above)
Comment (BrunoQuaresma) [reply site/tailwind.config.js:12]: It wasn't working until I added it. Can you test it on your end to double-check?

**Comment by BrunoQuaresma** [comment]
> @BrunoQuaresma just to confirm since there is no description. HSL is needed to get the alpha for colors to work correctly?

My bad 🙏 You are correct!

**Comment by jaaydenh** [comment]
@BrunoQuaresma I originally made a mistake putting fontSize under theme instead of theme->extend. the former overwrites all the existing fontSize classes. I moved this to extend so it only overrides what is explicitly added in the config. So `text-sm` should work correctly now.

**Review by jaaydenh — APPROVED** [review]
LGTM

**Pull request merged**